x-pack/platform/plugins/shared/streams/server/routes/internal/streams/processing/suggestions_handler.eval.ts

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-console */
+
+/**
+ * Add LogsHub sample data using:
+ *
+ * ```bash
+ * node scripts/synthtrace.js sample_logs --live --liveBucketSize=1000
+ * ```
+ *
+ * Run evaluation script using:
+ *
+ * ```bash
+ * yarn run ts-node --transpile-only x-pack/platform/plugins/shared/streams/server/routes/internal/streams/processing/suggestions_handler.eval.ts
+ * ```
+ */
+
+import { Client } from '@elastic/elasticsearch';
```

Comment:
Do you think that this file might be a good candidate for the `scripts` folder that some plugins have? Example for APM https://github.com/elastic/kibana/tree/main/x-pack/solutions/observability/plugins/apm/scripts